src/store/active.js

```diff
@@ -1,57 +1,44 @@
-import {
-  SET_ACTIVE_MEDIA_ITEM,
-  PAUSE_ACTIVE_MEDIA_ITEM,
-  EJECT_ACTIVE_MEDIA_ITEM,
-  SET_MESSAGE,
-} from '~/constants/mutation-types'
+import { defineStore } from 'pinia'
+
+export const ACTIVE = 'active'
```

Comment:
Yeah, this isn't necessary. Since we should name the store for dev-tools, I assumed using a constant (especially the old one for continuity) would be nice. But this doesn't need to be exported.